docs/_content/overige/technisch/zds-vs-zgw.md

```diff
@@ -7,27 +7,22 @@ De Zaak- en Documentservices (ZDS) wijken af van de Zaakgericht werken (ZGW)
 APIs, maar er is ook overlap. De belangrijkste verschillen en overeenkomsten op
 een rij:
 
-1. De ZGW APIs zijn in REST/JSON terwijl ZDS volgens SOAP/XML werken.
-2. De ZGW APIs zijn veel flexibeler dan ZDS. Dit is nodig omdat er geen directe
-   acties meer op de database mag worden gedaan door bijvoorbeeld een 
-   Zaaksysteem. Alle acties moeten via de ZGW APIs verlopen.
-3. De ZGW APIs kunnen alles wat ZDS ook kan.
-4. De ZGW APIs zijn opgedeeld. Er is niet meer 1 specificatie of endpoint om
-   alle operaties, zoals deze in ZDS bestonden, uit te voeren of data op te
-   vragen.
-5. De ZGW APIs zullen sneller opvolgende versies krijgen. Hierbij is het niet
-   ondenkbaar dat de Zaken API op versie 2 zit, terwijl de Besluiten API al op
-   versie 4 zit. Het is daarom ook niet zo dat ZGW een versie nummer krijgt.
-   Immers is ZGW alleen maar de verzamelnaam van alle Zaakgericht werken APIs. 
-6. De ZGW APIs worden aangetoond middels een referentie implementatie om de
-   theoretische specificaties te toetsen aan de praktijk, migratie problemen
-   te detecteren en om het vereiste gedrag te definieren daar waar de 
-   specificatie ruimte laat voor interpretaties.
-7. De ZGW APIs zijn ontwikkeld vanuit developer-first oogpunt waardoor
-   koppelingen sneller gemaakt kunnen worden.
-8. De ZGW APIs kennen alleen synchrone operaties.
-9. De ZGW APIs worden net als ZDS beheerd en onderhouden door VNG, zij het in
-   een andere vorm.
+[Zaak-Documentservices](https://www.gemmaonline.nl/index.php/Zaak-_en_Documentservices) | [ZGW APIs](https://ref.tst.vng.cloud/standaard/index)\*
+--- | ---
+SOAP/XML gebaseerd	                                                                        | REST/JSON gebaseerd
```

Comment:
Dit is dus de achterliggende techniek en het (standaard) formaat.

Reply:
begrijp ik goed dat je liever een aantal paragrafen met daaronder vergelijkingen tussen ZDS en ZGW API's op dat vlak ziet?

Reply:
Ja, maar ik weet niet of het dus persé beter wordt, maar ik vind de opzet van de tabel niet duidelijk geworden.

Reply:
Voor een goed vergelijk moet het verschil tussen de onderliggende architecturen van de beide standaarden uiteen gezet worden. ZDS veronderstelt synchroniseren, micro services gaat uit van gegevens bij de bron. Zoals Andy gister ook terecht opmerkte zijn de ZGW API's niet een directe vervanger van ZDS, maar veronderstelt een andere architectuur van applicaties en het gegevenslandschap. Daar moeten we iets over zeggen. Dat kan in dit document. Maar ik zal dat ook in de productvisie laten landen.